handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -174,12 +174,11 @@ func getClaim(t *jwt.Token, claim string) (interface{}, bool) {
 	if claims, ok := t.Claims.(jwt.MapClaims); ok {
 		c, has := claims[claim]
 		if !has {
-			log.Printf("Missing claim %q for token %v", claim, maskToken(t.Raw))
 			return "", false
 		}
 		return c, true
 	}
-	log.Printf("Missing claim %q for token", claim)
+	log.Printf("Missing claim %q for token %v", claim, maskToken(t.Raw))	
```

Comment:
I am not the right guy to ask for the requirements that lead to the implementation of this line back in the days. I just corrected it. If you say it can be safely removed I'll do so.